everest.yaml

```diff
@@ -1,6 +1,6 @@
 - Name: CommunalHelper
   Version: 0.0.0-dev
-  DLL: src/bin/Debug/net452/CommunalHelper.dll
+  DLL: src/bin/Debug/net7.0/CommunalHelper.dll
   Dependencies:
     - Name: Everest
```

Comment:
Might need to change dependency name to `EverestCore`?

Reply:
Right my bad, just [looked at](https://github.com/EverestAPI/CelesteModTemplate/blob/main/CelesteMod/everest-core.yaml) I'll update it and test that it works